AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/routes/v1/TokenRoutes.java

```diff
@@ -62,6 +63,9 @@ public TokenResponseApiModel createToken(Request req, Response res) {
     private TokenResponseApiModel generateToken(User user) {
         // Use the user_id as the subject for the issued token
         TokenIssuer.IssuedToken issuedToken = tokenIssuer.generateToken(Integer.toString(user.getUserId()));
-        return new TokenResponseApiModel(issuedToken.getToken(), issuedToken.getExpiryInstant().toString(), user.getUserId(), user.isAdmin());
+        int userId = user.getUserId();
+        boolean isAdmin = user.isAdmin();
+        Integer agentId = user.getAgentId();
```

Comment:
Similarly here - we only pulled this out for debugging reason, so it might be nice to clean this back into the one line